src/box/lua/iproto.c

```diff
@@ -117,8 +101,119 @@ push_iproto_key_enum(struct lua_State *L)
 		{"EVENT_DATA", IPROTO_EVENT_DATA},
 		{"TXN_ISOLATION", IPROTO_TXN_ISOLATION},
 		{"VCLOCK_SYNC", IPROTO_VCLOCK_SYNC},
+};
+
+static const char *const iproto_key_aliases[] = {
+	"request_type",
```

Comment:
Redefining all constants in lower-case manually isn't very convenient. Better lower case them automatically as you used to.

Reply:
I don't want dynamic allocation.

Reply:
Duplicating constants in lower and upper case doesn't look good to me. Up to @alyapunov.

Reply:
We should not duplicate data, that increases latter support cost. On the other hand I see no problem here to construct (even allocated) those strings at runtime.

Of course I would prefer compile-time generated lowercase names. Or even case-insensitive hash table that is used for translation. But in plain C the best way is to duplicate those string in some `*_init` function.

Reply:
Fixed: now all strings for translation table are dynamically allocated.